app/components/Card.tsx

```diff
@@ -1,19 +1,22 @@
 import { tx } from '@helpwave/common/twind/index'
 import type { PropsWithChildren } from 'react'
+import type { Class } from '@twind/core'
 
 export type CardProps = {
   isSelected?: boolean,
-  onTileClicked?: () => void
+  onTileClicked?: () => void,
+  classes?: Class[]
 }
 
 export const Card = ({
   children,
   isSelected = false,
-  onTileClicked = () => undefined
+  onTileClicked = () => undefined,
+  classes = [],
 }: PropsWithChildren<CardProps>) => {
   return (
     <div onClick={onTileClicked}
-         className={tx('cursor-pointer rounded-md py-2 px-4 border border-2 hover:border-hw-primary-700 w-full', { 'border-hw-primary-700': isSelected })}>
+         className={tx('cursor-pointer rounded-md py-2 px-4 border border-2 hover:border-hw-primary-700 w-full', { 'border-hw-primary-700': isSelected }, ...classes)}>
```

Comment:
I'm not always that much of a fan of exposing `className` to the outside

Regardless, I think this should just be a `string`, if the consumer of this component wants to use twind/tailwind classes they should call `tw` or `tx` themself and pass the return value of that along

Reply:
But why are you not a fan of exposing `className`? What are the downsides?

Reply:
Exposing `classNames` often leads to hacky and diverging styling (w.r.t the original design) if used in different places and gets overused quite a lot. If you want some special style you should think really hard about it and maybe add a variant to the original component.

There are of course occasions where exposing this makes a lot of sense but it gets used too much and without much thought too often.

Reply:
@MaxSchaefer 
(pinging you as this got marked as resolved and I'm not sure if you'd get notified about my answer without a ping)